src/ImagePotentialBondGPU.h

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2018-2020, Michael P. Howard
+// Copyright (c) 2021-2025, Auburn University
+// Part of azplugins, released under the BSD 3-Clause License.
+
```

Comment:
Probably all the `ImagePotentialBondGPU` files should have comments for the license along the lines of what you put in the CPU one saying we copied this from HOOMD.